src/trackPromise.js

```diff
@@ -15,10 +15,11 @@ export const trackPromise = (promise, area) => {
   const promiseInProgress = anyPromiseInProgress(area);
   emitter.emit(promiseCounterUpdateEventId, promiseInProgress, area);
 
-  promise
-    .then(() => decrementPromiseCounter(area),
-      () => decrementPromiseCounter(area)
-    );
+  const onResolveHandler = (arg) => {
```

Comment:
Does it resolve the issue?

Why is it working on 1.0.2 version and not in 1.1.0? Seeing last PR the code is the same except the area parameter

Reply:
No it doesn't. The issue in 1.1.0 is not related with this enhancement, but while investigating it I came across with this improvement. I will soon release a fix for the issue. We are discussing different approaches.